src/allmydata/test/web/test_root.py

```diff
@@ -138,3 +149,76 @@ def __init__(self):
 
         self.assertThat(item.slotData.get("version"), Equals(""))
         self.assertThat(item.slotData.get("nickname"), Equals(""))
+
+
+class RenderRoot(AsyncTestCase):
+
+    @inlineCallbacks
+    def test_root_json(self):
```

Comment:
Does this actually reproduce the problem when run without the fix? Seems like you would need to have two servers, one with availability, one without. If I'm correct, you should do the "make sure the test actually fail when fix is commented out" dance.

Reply:
Thanks, I'll double-check (I did confirm this branch works with magic-folders but don't recall if I confirmed the utest ;)

Reply:
indeed it did not, thanks!